benchmark/nixlbench/contrib/Dockerfile

```
@@ -280,7 +280,7 @@ RUN ls -ll /workspace/nixlbench
 RUN rm -rf build && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)